molo/yourwords/models.py

```diff
@@ -0,0 +1,26 @@
+from wagtail.wagtailcore.models import Page
+
+
+class Competition(Page):
+    name = Page.CharField(max_length=128)
+    description = Page.TextField()
+    content = Page.TextField()
```

Comment:
Please use the [StreamField](https://github.com/praekelt/molo/blob/develop/molo/core/models.py#L267-L274) what we use in the ArticlePage for this field